翻译主菜单 (#1816)
cocos-creator/creator-docs: en/editor/mainMenu/index.md

> # Main Menu

Author: @slackmoehrle Have you reviewed this one?

Reviewer: yes

| 3D Objects | Creator provides some relatively basic static model controls for use, currently including **Cube**, **Cylinder**, **Sphere**, **Capsule**, **Cone**, **Torus**, **Flat** and **Quad**. To create other types of models, refer to the [MeshRenderer component](../../engine/renderable/model-component.md) documentation. |
| 2D Objects | Create Creator's pre-defined 2D nodes in the scene containing the base rendering components, which currently include **Graphics**, **Label**, **Mask**, **ParticleSystem2D**, **Sprite**, **SpriteSplash** (monochrome) and **TiledMap** (map), for details please refer to the [Introduction to 2D Rendering Components](../../ui-system/components/editor/render-component.md) documentation. |
| UI Components | Create Creator's pre-defined nodes containing basic UI components in the scene, which currently include Common UI controls such as **Button**, **Widget**, **Layout**, **ScrollView**, **EditBox** nodes, etc. For more UI components, please refer to the [UI Basic Components](../../ui-system/components/editor/base-component.md) documentation. |

> | Light | Create Creator preset nodes in the scene containing base light components, currently including **Parallel Light**, **Spherical Light** and **Spot Light**, please refer to the [Light](../../concepts/scene/light.md) documentation. |

Reviewer 2: ```suggestion
| Light | Create Creator preset nodes in the scene containing base light components, currently including **Parallel Light**, **Spherical Light** and **Spot Light**, please refer to the [Lighting](../../concepts/scene/light.md) documentation. |
```

| Effects | Create a Creator preset in the scene containing [Particle System](../../particle-system/overview.md) component in the scene. More effects components can be added in the **Property Inspector** panel by clicking **Add Component -> Special Effects**. |

> | Camera | Creates a node in the scene with the Camera component pre-defined by the Creator. For details on how to use it, please refer to the [Camera Components](../components/camera-component.md) documentation. |

Reviewer 2: ```suggestion
| Camera | Creates a node in the scene with the Camera component pre-defined by the Creator. For details on how to use it, please refer to the [Camera component](../components/camera-component.md) documentation. |
```

| Terrain | Creates a node containing the terrain component pre-defined by the Creator in the scene. Please refer to the [Terrain System](../terrain/index.md) documentation for more details on how to use it. |

## Project

This option is mainly used to perform preview run build projects, project configuration and custom builds, etc.

![project](./img/project.png)

| Option | Description |
| :--- | :--- |
| Project Settings | Opens [Project Settings](../project/index.md) panel to set the relevant configuration options for a specific project. |

> | Lightmapping | Opens the [Lightmapping](../../concepts/scene/light/lightmap.md) panel to configure the baking parameters for generating light maps. |

Author: ```suggestion
| Lightmapping | Opens the [Lightmapping](../../concepts/scene/light/lightmap.md) panel to configure the baking properties for generating light maps. |
```

| Create Extension | This option is used to generate an [extension package](../extension/first.md) in the project/global directory to extend the editor with features. |

## Developer

This option contains mainly development-related menu functions such as scripts, engine and DevTools.

![developer](./img/developer.png)

| Option | Description |
| :--- | :--- |
| Compile Engine (<kbd>Ctrl</kbd>/<kbd>Cmd</kbd> + <kbd>F7</kbd>) | Compile custom engine, please refer to the [Engine Customization Workflow](../../advanced-topics/engine-customization.md) documentation. |
| Rebuild Native Engine | Compile the custom native engine simulator, please refer to [Customizing the Native Engine Simulator](../../advanced-topics/engine-customization.md#modifying-native-engine-simulator). |
| Message DevTools | Opens the Message DevTool for debugging IPC interactions at runtime inside the editor. |
| Tester | The editor's built-in extended testing tool, not yet fully functional |
|Toggle Graphics Tool | Toggles the Graphics tool panel for debugging scene rendering. |

> | Message Manager | Opens the [Message Manager](../extension/contributions-messages.md) panel to display public messages and their descriptions for each feature defined by the editor. |

Reviewer 2: ```suggestion
| Message List | Opens the [Message Manager](../extension/contributions-messages.md) panel to display public messages and their descriptions for each feature defined by the editor. |
```

Author: 这个确实是Message Manager啊
<img width="309" alt="截屏2021-09-29 下午4 49 52" src="https://user-images.githubusercontent.com/55615178/135235587-1aaf8600-8613-4b92-b10c-76d395eaf6b4.png">

| UI Components | Opens the UI Components panel, which lists how to use the pre-defined UI components provided within the editor, please refer to [UI Components](../extension/ui.md). |
| VS Code Workflow | The working environment-related features of the VS Code editor, currently supporting **Add Chrome debug configuration** and **Add compile task**. |
| Toggle DevTools (<kbd>Ctrl</kbd>/<kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>I</kbd>) | Opens DevTools window for editor interface extension development. It is also possible to customize the log output to **Console**, please refer to [Custom output messages](../console/index.md#custom-output-messages). |
| Open Assets DevTools | Opens the Assets DevTool panel for viewing log messages during modifications to the asset-db process. |
| Open Scene DevTools | Opens the Scene DevTool panel to view log messages during modifications to the scene. |
| Open Build DevTools | Opens the Build DevTool to view all log messages generated during the build process including the call stack. |

## Help

![help](./img/help.png)

| Option | Description |
| :--- | :--- |

> | User Manual | Open [User Manual Documentation](https://docs.cocos.com/creator/3.0/manual/en/) in default browser. |

Reviewer 2: ```suggestion
| User Manual | Open [User Manual Documentation](../../index.md) in default browser. |
```

> | API Documentation | Open [API Reference Documentation](__APIDOC__/en/) in default browser. |

Reviewer 2: ```suggestion
| API Reference | Open [API Reference Documentation](__APIDOC__/en/) in default browser. |
```

> | Forum | Open [Cocos Forum](https://forum.cocos.org/c/58) in default browser. |

Reviewer 2: ```suggestion
| Forum | Open [Cocos Creator Forum](https://forum.cocos.org/c/58) in default browser. |
```

> | Forums | Open [Cocos Creator Forums](https://forum.cocos.org/c/58) in default browser. |

Reviewer 2: 为啥论坛后面要加 s

Author: 我是看官网上也是加了s，那我删了吧

> | Forums | Open [Cocos Creator Forums](https://forum.cocos.org/c/58) in default browser. |

Author: ```suggestion
| Forum | Open [Cocos Forum](https://forum.cocos.org/c/58) in default browser. |
```

> | Update Log | Open [Cocos Creator Version Update Log](https://www.cocos.com/creator) in default browser. |

Reviewer 2: ```suggestion
| Release Notes | Open the [release notes](https://www.cocos.com/creator) for each version of Cocos Creator in default browser. |
```
